src/NServiceBus.Transport.SQS/Configure/SqsTransportInfrastructure.cs

```diff
@@ -98,7 +98,8 @@ QueueCreator CreateQueueCreator()
 
         MessageDispatcher CreateMessageDispatcher()
         {
-            return new MessageDispatcher(configuration, s3Client, sqsClient, snsClient, queueCache, topicCache);
+            var doNotWrapOutgoingMessages = settings.GetOrDefault<bool>(SettingsKeys.DoNotWrapOutgoingMessages);
+            return new MessageDispatcher(configuration, s3Client, sqsClient, snsClient, queueCache, topicCache, !doNotWrapOutgoingMessages);
```

Comment:
The not here makes the code harder to understand. What was the reason for using it?

Reply:
This is the same as on master. Since the default behaviour is to wrap the messages, the setting introduced is doNotWrapOutgoingMessage. In code we want to deal with the opposite of that (ie wrapOutgoingMessages) so when we pass it into the dispatcher we switch it over to that.

Reply:
:exploding_head: But yeah it is also like that on master. Discussed with @mauroservienti and resolved it.